boost-common/src/main/java/io/openliberty/boost/utils/LibertyServerConfigGenerator.java

```diff
@@ -215,4 +215,13 @@ public void addBootstrapProperties(Properties properties) throws IOException{
               bootstrapProperties.put(key, value);
         }
     }
+    
+    public void addConfigForApp(String artifactId, String version){
+    	Element appCfg = doc.createElement(APPLICATION);
+    	appCfg.setAttribute("context-root", "/");
+    	appCfg.setAttribute("location", artifactId+"-"+version+".war");
```

Comment:
Is it safe to assume that we'll always be building a WAR? Should we get the packaging type dynamically instead?